Package.swift

```diff
@@ -229,11 +229,13 @@ let package = Package(
 // package dependency like this but there is no other good way of expressing
 // this right now.
 
+/// When not using local dependencies, the branch to use for llbuild and TSC repositories
+let relatedDependenciesBranch = ProcessInfo.processInfo.environment["SWIFTPM_DEPS_BRANCH"] ?? "release/5.3"
```

Comment:
thanks for putting this together @yonihemi, we discussed more and think the env variable as defined here is not adding much value. lets just set the variable to the target branch string (ie `et relatedDependenciesBranch = "release/5.3"`) and open similar PR for the main branch